roachtest: sqlalchemy failed

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7330134?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7330134?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fc7a9c040029eeac0791db653f1d13a3cbda4e76](https://github.com/cockroachdb/cockroach/commits/fc7a9c040029eeac0791db653f1d13a3cbda4e76):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-659-gfc7a9c0400
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21217

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7342327?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7342327?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [1c30a38d67eb6a9fea51082937de24cb15c1eead](https://github.com/cockroachdb/cockroach/commits/1c30a38d67eb6a9fea51082937de24cb15c1eead):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-666-g1c30a38d67
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master release-blocker]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0 release-blocker]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7354437?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7354437?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [56e0cae2dd496aee737f273f6b0a77faae4f76bd](https://github.com/cockroachdb/cockroach/commits/56e0cae2dd496aee737f273f6b0a77faae4f76bd):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-670-g56e0cae2dd
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master release-blocker]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0 release-blocker]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377876?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377876?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [9048c36c4f4e851bfabde9209083fadeee7aa532](https://github.com/cockroachdb/cockroach/commits/9048c36c4f4e851bfabde9209083fadeee7aa532):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-683-g9048c36c4f
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7402587?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7402587?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [a91533a57a3bbb920b6e6b3bdb2c2824d7b0996e](https://github.com/cockroachdb/cockroach/commits/a91533a57a3bbb920b6e6b3bdb2c2824d7b0996e):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-706-ga91533a57a
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7423847?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7423847?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [9a72dc474bd00f1f28089ac217794015dd351190](https://github.com/cockroachdb/cockroach/commits/9a72dc474bd00f1f28089ac217794015dd351190):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-727-g9a72dc474b
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7466987?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7466987?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [9ec2ddb8dc118ce91bf806612bc4c0f86dd7b32b](https://github.com/cockroachdb/cockroach/commits/9ec2ddb8dc118ce91bf806612bc4c0f86dd7b32b):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-765-g9ec2ddb8dc
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7478539?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7478539?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [9ec2ddb8dc118ce91bf806612bc4c0f86dd7b32b](https://github.com/cockroachdb/cockroach/commits/9ec2ddb8dc118ce91bf806612bc4c0f86dd7b32b):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-765-g9ec2ddb8dc
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7490771?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7490771?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [9ec2ddb8dc118ce91bf806612bc4c0f86dd7b32b](https://github.com/cockroachdb/cockroach/commits/9ec2ddb8dc118ce91bf806612bc4c0f86dd7b32b):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-765-g9ec2ddb8dc
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7512780?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7512780?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [56072375c3e6184a2ac0494ce90b34a4bf0e9f45](https://github.com/cockroachdb/cockroach/commits/56072375c3e6184a2ac0494ce90b34a4bf0e9f45):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-787-g56072375c3
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7534926?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7534926?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [01b87663ef4dc69bbff90cea367526c67f71f84b](https://github.com/cockroachdb/cockroach/commits/01b87663ef4dc69bbff90cea367526c67f71f84b):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-810-g01b87663ef
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7555571?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7555571?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [42084db6b92ae7964e8b05033e238d4ddec33d1a](https://github.com/cockroachdb/cockroach/commits/42084db6b92ae7964e8b05033e238d4ddec33d1a):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-830-g42084db6b9
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92060 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7574307?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7574307?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [b430d1535e93215e0f01cb92171de214be0eba4d](https://github.com/cockroachdb/cockroach/commits/b430d1535e93215e0f01cb92171de214be0eba4d):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-842-gb430d1535e
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92060 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7593526?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7593526?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [022d19ab6c1c6f6dd477d5120b4c47a46f744238](https://github.com/cockroachdb/cockroach/commits/022d19ab6c1c6f6dd477d5120b4c47a46f744238):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-855-g022d19ab6c
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92060 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7605151?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7605151?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [bf7c1cf83275be4645fc77581be2cb9d5e8502c4](https://github.com/cockroachdb/cockroach/commits/bf7c1cf83275be4645fc77581be2cb9d5e8502c4):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-857-gbf7c1cf832
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92060 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7617021?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7617021?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [bf7c1cf83275be4645fc77581be2cb9d5e8502c4](https://github.com/cockroachdb/cockroach/commits/bf7c1cf83275be4645fc77581be2cb9d5e8502c4):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-857-gbf7c1cf832
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92060 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7636771?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7636771?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [2a71a5cca1f66313fd39956f1a3db070f67ade15](https://github.com/cockroachdb/cockroach/commits/2a71a5cca1f66313fd39956f1a3db070f67ade15):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-868-g2a71a5cca1
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92060 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7655691?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7655691?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [f660a454f6781a379fcc2a17646dc859ac5af997](https://github.com/cockroachdb/cockroach/commits/f660a454f6781a379fcc2a17646dc859ac5af997):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-875-gf660a454f6
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7672317?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7672317?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fe3ec49ff917b9b0eab2a4048fec31d56fc26daf](https://github.com/cockroachdb/cockroach/commits/fe3ec49ff917b9b0eab2a4048fec31d56fc26daf):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-885-gfe3ec49ff9
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7685385?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7685385?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fd133ba3af3954c93584a501cf68dfa3131e586c](https://github.com/cockroachdb/cockroach/commits/fd133ba3af3954c93584a501cf68dfa3131e586c):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-887-gfd133ba3af
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7697798?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7697798?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fd133ba3af3954c93584a501cf68dfa3131e586c](https://github.com/cockroachdb/cockroach/commits/fd133ba3af3954c93584a501cf68dfa3131e586c):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-887-gfd133ba3af
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7709113?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7709113?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fd133ba3af3954c93584a501cf68dfa3131e586c](https://github.com/cockroachdb/cockroach/commits/fd133ba3af3954c93584a501cf68dfa3131e586c):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-887-gfd133ba3af
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7720748?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7720748?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fd133ba3af3954c93584a501cf68dfa3131e586c](https://github.com/cockroachdb/cockroach/commits/fd133ba3af3954c93584a501cf68dfa3131e586c):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-887-gfd133ba3af
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7742787?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7742787?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [0ebce067f89a997f1daf04b4180d4dc5035c4fea](https://github.com/cockroachdb/cockroach/commits/0ebce067f89a997f1daf04b4180d4dc5035c4fea):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-907-g0ebce067f8
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7763956?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7763956?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [d978988afcfc78c02c23a3836ba90e4a4faeeb25](https://github.com/cockroachdb/cockroach/commits/d978988afcfc78c02c23a3836ba90e4a4faeeb25):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-917-gd978988afc
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7782685?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7782685?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ca673993473629a46370100108dd8c185a592e91](https://github.com/cockroachdb/cockroach/commits/ca673993473629a46370100108dd8c185a592e91):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-943-gca67399347
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7815219?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7815219?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ddca6ce770299653164c0c04fcf753af1c3069ca](https://github.com/cockroachdb/cockroach/commits/ddca6ce770299653164c0c04fcf753af1c3069ca):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-981-gddca6ce770
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92987 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7825003?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7825003?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [b8353892e5198e402456d54d5c41297d3a9287aa](https://github.com/cockroachdb/cockroach/commits/b8353892e5198e402456d54d5c41297d3a9287aa):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-983-gb8353892e5
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92987 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7834232?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7834232?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [b8353892e5198e402456d54d5c41297d3a9287aa](https://github.com/cockroachdb/cockroach/commits/b8353892e5198e402456d54d5c41297d3a9287aa):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-983-gb8353892e5
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92987 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7852350?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7852350?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [adf24a585553397a7ad8e4bb6da0609bb5d2e930](https://github.com/cockroachdb/cockroach/commits/adf24a585553397a7ad8e4bb6da0609bb5d2e930):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-999-gadf24a5855
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7867074?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7867074?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [208b1d3ad46811dcf527ff53636f47b040ff62a5](https://github.com/cockroachdb/cockroach/commits/208b1d3ad46811dcf527ff53636f47b040ff62a5):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1001-g208b1d3ad4
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93114 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7879926?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7879926?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [163fc1d72bb1e66344309c65dd497156694b72c4](https://github.com/cockroachdb/cockroach/commits/163fc1d72bb1e66344309c65dd497156694b72c4):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1010-g163fc1d72b
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7895050?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7895050?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [45cceae6a1277ab1970d24806cb57230814bf494](https://github.com/cockroachdb/cockroach/commits/45cceae6a1277ab1970d24806cb57230814bf494):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1020-g45cceae6a1
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7909446?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7909446?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [97d5489387634dfd25462c01cace49445d5698de](https://github.com/cockroachdb/cockroach/commits/97d5489387634dfd25462c01cace49445d5698de):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1026-g97d5489387
		Tests run against sqlalchemy rel_1_4_26
		371 Total Tests Run
		365 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7916385?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7916385?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ff69f981a02ea6906d4be138189c4531a6a420d1](https://github.com/cockroachdb/cockroach/commits/ff69f981a02ea6906d4be138189c4531a6a420d1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1032-gff69f981a0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93382 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7923334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7923334?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ff69f981a02ea6906d4be138189c4531a6a420d1](https://github.com/cockroachdb/cockroach/commits/ff69f981a02ea6906d4be138189c4531a6a420d1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1032-gff69f981a0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93382 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7937065?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7937065?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [c50f15fae7e62a9b5ad905e88314f69eb46c3e5c](https://github.com/cockroachdb/cockroach/commits/c50f15fae7e62a9b5ad905e88314f69eb46c3e5c):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1043-gc50f15fae7
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93382 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7952917?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7952917?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [bd54db769eed3e1f2a91cf63fcbb2d36182f0901](https://github.com/cockroachdb/cockroach/commits/bd54db769eed3e1f2a91cf63fcbb2d36182f0901):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1064-gbd54db769e
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93382 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91349 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.2.0]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7969285?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7969285?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [cab302aa7adeaf41f45b7b6532d9f72bb0d81b6e](https://github.com/cockroachdb/cockroach/commits/cab302aa7adeaf41f45b7b6532d9f72bb0d81b6e):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1081-gcab302aa7a
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93382 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7985142?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7985142?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [d6abb72b4f5b5fca4deac73c3b0f80c6560e8c1b](https://github.com/cockroachdb/cockroach/commits/d6abb72b4f5b5fca4deac73c3b0f80c6560e8c1b):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1093-gd6abb72b4f
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #93382 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8000047?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8000047?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [a9e82c38ce99ee86eb160df5fa52e1e5e818924c](https://github.com/cockroachdb/cockroach/commits/a9e82c38ce99ee86eb160df5fa52e1e5e818924c):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1105-ga9e82c38ce
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8006764?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8006764?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [e7221ee3f22db90628f23e0c6d8754780b2b6188](https://github.com/cockroachdb/cockroach/commits/e7221ee3f22db90628f23e0c6d8754780b2b6188):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1107-ge7221ee3f2
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8014322?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8014322?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [cf1e7e6bc6ef9e55510b9ed13bd068bb3894cd92](https://github.com/cockroachdb/cockroach/commits/cf1e7e6bc6ef9e55510b9ed13bd068bb3894cd92):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1111-gcf1e7e6bc6
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8031009?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8031009?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [fc04d8e1eabf6b58d32e9792fd11f97c493480ad](https://github.com/cockroachdb/cockroach/commits/fc04d8e1eabf6b58d32e9792fd11f97c493480ad):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1136-gfc04d8e1ea
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8046119?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8046119?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [773649a2b08605ba2031dce9b3c8858b15803ca2](https://github.com/cockroachdb/cockroach/commits/773649a2b08605ba2031dce9b3c8858b15803ca2):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1148-g773649a2b0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8061599?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8061599?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [e528c6976451fa44b270364b18ec1107a49b6fc6](https://github.com/cockroachdb/cockroach/commits/e528c6976451fa44b270364b18ec1107a49b6fc6):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.0-beta.2-1162-ge528c69764
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #94049 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-21.2 release-blocker]
- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8078837?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8078837?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [8ff653b220128d32bbd7eff2717d6212b2b6ac12](https://github.com/cockroachdb/cockroach/commits/8ff653b220128d32bbd7eff2717d6212b2b6ac12):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-69-g8ff653b220
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8087955?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8087955?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ea0e966fe05a788fd123a08cec8ef5e935fa5496](https://github.com/cockroachdb/cockroach/commits/ea0e966fe05a788fd123a08cec8ef5e935fa5496):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-76-gea0e966fe0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8094323?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8094323?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ea0e966fe05a788fd123a08cec8ef5e935fa5496](https://github.com/cockroachdb/cockroach/commits/ea0e966fe05a788fd123a08cec8ef5e935fa5496):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-76-gea0e966fe0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8100547?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8100547?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ea0e966fe05a788fd123a08cec8ef5e935fa5496](https://github.com/cockroachdb/cockroach/commits/ea0e966fe05a788fd123a08cec8ef5e935fa5496):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-76-gea0e966fe0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8107025?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8107025?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [ea0e966fe05a788fd123a08cec8ef5e935fa5496](https://github.com/cockroachdb/cockroach/commits/ea0e966fe05a788fd123a08cec8ef5e935fa5496):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-76-gea0e966fe0
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8117105?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8117105?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [6d4dd6df63c2c9d516153404fd15d62d4e3c9908](https://github.com/cockroachdb/cockroach/commits/6d4dd6df63c2c9d516153404fd15d62d4e3c9908):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-82-g6d4dd6df63
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8127845?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8127845?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [afab335620e28c5d8c849ba09e50ec12c2836b3f](https://github.com/cockroachdb/cockroach/commits/afab335620e28c5d8c849ba09e50ec12c2836b3f):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
	orm_helpers.go:191,orm_helpers.go:117,sqlalchemy.go:239,sqlalchemy.go:47,test_runner.go:930: 
		Tests run on Cockroach v22.2.1-84-gafab335620
		Tests run against sqlalchemy rel_1_4_26
		419 Total Tests Run
		413 tests passed
		6 tests failed
		195 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
		--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
		For a full summary look at the sqlalchemy artifacts 
		
		An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8138886?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8138886?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1](https://github.com/cockroachdb/cockroach/commits/07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
(test_impl.go:292).Fatalf: 
Tests run on Cockroach v22.2.1-108-g07a53a3660
Tests run against sqlalchemy rel_1_4_26
419 Total Tests Run
413 tests passed
6 tests failed
195 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
For a full summary look at the sqlalchemy artifacts
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8147282?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8147282?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1](https://github.com/cockroachdb/cockroach/commits/07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
(test_impl.go:292).Fatalf: 
Tests run on Cockroach v22.2.1-108-g07a53a3660
Tests run against sqlalchemy rel_1_4_26
419 Total Tests Run
413 tests passed
6 tests failed
195 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
For a full summary look at the sqlalchemy artifacts
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8153431?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8153431?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1](https://github.com/cockroachdb/cockroach/commits/07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
(test_impl.go:292).Fatalf: 
Tests run on Cockroach v22.2.1-108-g07a53a3660
Tests run against sqlalchemy rel_1_4_26
419 Total Tests Run
413 tests passed
6 tests failed
195 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
For a full summary look at the sqlalchemy artifacts
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8159354?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8159354?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1](https://github.com/cockroachdb/cockroach/commits/07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
(test_impl.go:292).Fatalf: 
Tests run on Cockroach v22.2.1-108-g07a53a3660
Tests run against sqlalchemy rel_1_4_26
419 Total Tests Run
413 tests passed
6 tests failed
195 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
For a full summary look at the sqlalchemy artifacts
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.sqlalchemy [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8166131?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8166131?buildTab=artifacts#/sqlalchemy) on release-22.2 @ [07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1](https://github.com/cockroachdb/cockroach/commits/07a53a36601e9ca5fcffcff55f69b43c6dfbf1c1):


```
test artifacts and logs in: /artifacts/sqlalchemy/run_1
(test_impl.go:292).Fatalf: 
Tests run on Cockroach v22.2.1-108-g07a53a3660
Tests run against sqlalchemy rel_1_4_26
419 Total Tests Run
413 tests passed
6 tests failed
195 tests skipped
0 tests ignored
0 tests passed unexpectedly
6 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_view (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_temp_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_view_schema (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table (unexpected)
--- FAIL: test/test_suite_sqlalchemy.py::HasTableTest_cockroachdb+psycopg2_9_5_0::test_has_table_schema (unexpected)
For a full summary look at the sqlalchemy artifacts
An updated blocklist (sqlAlchemyBlocklist) is available in the artifacts' sqlalchemy log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #91351 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-master]
- #91343 roachtest: sqlalchemy failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlalchemy.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/asyncpg.go
+++ b/pkg/cmd/roachtest/tests/asyncpg.go
@@ -91,7 +91,7 @@ func registerAsyncpg(r registry.Registry) {
 		}
 
 		if err := repeatRunE(
-			ctx, t, c, node, "create virtualenv", `virtualenv venv`,
+			ctx, t, c, node, "create virtualenv", `virtualenv --clear venv`,
 		); err != nil {
 			t.Fatal(err)
 		}

--- a/pkg/cmd/roachtest/tests/django.go
+++ b/pkg/cmd/roachtest/tests/django.go
@@ -94,7 +94,7 @@ func registerDjango(r registry.Registry) {
 		}
 
 		if err := repeatRunE(
-			ctx, t, c, node, "create virtualenv", `virtualenv venv`,
+			ctx, t, c, node, "create virtualenv", `virtualenv --clear venv`,
 		); err != nil {
 			t.Fatal(err)
 		}

--- a/pkg/cmd/roachtest/tests/sqlalchemy.go
+++ b/pkg/cmd/roachtest/tests/sqlalchemy.go
@@ -93,7 +93,7 @@ func runSQLAlchemy(ctx context.Context, t test.Test, c cluster.Cluster) {
 	}
 
 	if err := repeatRunE(
-		ctx, t, c, node, "create virtualenv", `virtualenv venv`,
+		ctx, t, c, node, "create virtualenv", `virtualenv --clear venv`,
 	); err != nil {
 		t.Fatal(err)
 	}

